### DISCUSSION TITLE
ScriptReference/AssetDatabase.CreateFolder

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#84] [Notes]
# ScriptReference/AssetDatabase.CreateFolder

https://docs.unity3d.com/ScriptReference/AssetDatabase.CreateFolder.html

**Comment by vertxxyz** [comment]
If the parent folder has a trailing slash, eg. `"Assets/"` then this function may silently fail.

**Comment by docscomments** [comment]
**Comment by `@ByronMayne`:**  

Unlike [`System.Directory.CreateDirectory`](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory), if the directory already exists Unity will create a second directory with a unique name by appending a number to the end.
